### DISCUSSION TITLE
救一下！！！

### DISCUSSION THREAD

**Discussion by Larksonguesinaspic** [share121/inter-knot#2112] [其他]
https://private-user-images.githubusercontent.com/66072099/357764734-3c86d461-a165-4d2d-92fa-58e8f0fbc4c1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM2MzQ3NTQsIm5iZiI6MTcyMzYzNDQ1NCwicGF0aCI6Ii82NjA3MjA5OS8zNTc3NjQ3MzQtM2M4NmQ0NjEtYTE2NS00ZDJkLTkyZmEtNThlOGYwZmJjNGMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDExMjA1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFkN2Y1N2JlNTFjYzkyMzMxODYxNWRiNzUwYjExMTYzZTU1YzFjZTY0NGNjMzI3ZTVhMTU1MGEzYmQ2MmYzM2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HPkeR_pEB5byNeBz_3y6Oq4z9Sm8bY-xKCBuWN-BJhQ
我是新艾利都市政一名普通的公务员，之前我领导投靠叛军被抓了，中午时黑曜石营的士兵把我们都抓了，我同事还被大记忆恢复术了！！
有人捞一下我吗？

**Comment by shydrogen** [comment]
这个委托我接了！<img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/889ec04140bf60811490c8d14a2ac327_6845707261677208256.png" alt="冲刺" width="75" >

**Comment by Maixiaofanqie** [comment]
黑曜石营真的是我们这种小喽啰惹得起的吗
